<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardTemplate</class>
 <widget class="QWidget" name="CardTemplate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1196</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="entreesButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Manjari</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>EntrÃ©es</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="platsButton">
      <property name="font">
       <font>
        <family>Manjari</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Plats</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="dessertsButton">
      <property name="font">
       <font>
        <family>Manjari</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Desserts</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="menusButton">
      <property name="font">
       <font>
        <family>Manjari</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Menus</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="boissonsButton">
      <property name="font">
       <font>
        <family>Manjari</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Boissons</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>901</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="rightLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
